action.yml

```diff
@@ -0,0 +1,202 @@
+---
+name: Steampipe Check
+author: Turbot
+description: Run Steampipe mod controls and create annotations for Infrastructure as Code controls.
+branding:
+  color: red
+  icon: shield
+inputs:
+  mod-url:
+    description: "Mod URL to clone. Examples: https://github.com/turbot/steampipe-mod-aws-compliance"
+    required: true
+  mod-branch:
+    description: "Mod branch or tag to clone. Defaults to 'main'. Examples: main, release/v0.7, 0.5, 1.0."
```

Comment:
Do the last 2 examples work, e.g., 0.5, 1.0 (assuming these are for tags)? Or do we need the v in front of them?

Reply:
If there are tags or branches with those specific names they would work, I'll prefix with `v` though as this is a more common pattern.